crawl-ref/source/webserver/server.py

```diff
@@ -18,7 +18,10 @@
 class MainHandler(tornado.web.RequestHandler):
     def get(self):
         host = self.request.host
-        if self.request.protocol == "https":
+        proto = self.request.headers.get_list('x-forwarded-proto')
+        if len(proto) > 0 : proto = proto[0]
```

Comment:
nit: the space after `:` is not like existing code style

Reply:
Whoops, this is my day job sneaking through